obspy/signal/polarization.py

```diff
@@ -359,12 +359,31 @@ def fit_func(beta, x):
     in_slope = out.beta[0]
     in_error = out.sd_beta[0]
 
-    azim = math.atan2(1.0, az_slope)
-    inc = math.atan2(1.0, in_slope)
-    az_error = 1.0 / ((1.0 ** 2 + az_slope ** 2) * azim) * az_error
-    in_error = 1.0 / ((1.0 ** 2 + in_slope ** 2) * inc) * in_error
+    azimuth = math.atan2(1.0, az_slope)
+    incidence = math.atan2(1.0, in_slope)
 
-    return math.degrees(azim), math.degrees(inc), az_error, in_error
+    az_error = 1.0 / ((1.0 ** 2 + az_slope ** 2) * azimuth) * az_error
+    # az_error = math.degrees(az_error)
+    in_error = 1.0 / ((1.0 ** 2 + in_slope ** 2) * incidence) * in_error
+    # in_error = math.degrees(in_error)
+
+    azimuth = math.degrees(azimuth)
+    incidence = math.degrees(incidence)
+
+    if azimuth < 0.0:
+        azimuth = 360.0 + azimuth
```

Comment:
Yeah, probably some not-so-nice looking code in here but this PR is more about the essential fix of the functionality. I would say we leave cosmetics to `master`.